src/components/table/pagination.tsx

```diff
@@ -0,0 +1,75 @@
+import ChevronLeft from '@geist-ui/react-icons/chevronLeft';
+import ChevronRight from '@geist-ui/react-icons/chevronRight';
+import React, { useEffect } from 'react';
```

Comment:
Could you use this pattern for consisitent? `import * as React from 'react';`

Reply:
ok